tests/Tests/RestControllers/PatientRestControllerTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace OpenEMR\Tests\RestControllers;
+
+use PHPUnit\Framework\TestCase;
+use OpenEMR\RestControllers\PatientRestController;
+use OpenEMR\Tests\Fixtures\FixtureManager;
+
+/**
+ * @coversDefaultClass OpenEMR\RestControllers\PatientRestController
+ */
+class PatientRestControllerTest extends TestCase
+{
+    const PATIENT_API_URL = "/apis/api/patient";
+
+    private $patientData;
+    private $patientController;
+    private $fixtureManager;
+
+    protected function setUp(): void
+    {
+        $this->patientData = array(
```

Comment:
Why are we not using "getSinglePatientFixture" ? Looking to learn 😄

Reply:
Ah! This may have been the result of the "cherry pick" merge I did or working on something late in the evening or early in the morning 😄 

I noticed that the patient data record has "all of the fields" that are supported by the GET API. I will update the fixture to include all of the fields, and then we can switch over to using the `getSinglePatientFixture`

Thanks!